nav2_dwb_controller/dwb_controller/src/dwb_controller.cpp

```diff
@@ -25,13 +25,19 @@ using nav2_tasks::TaskStatus;
 using dwb_core::DWBLocalPlanner;
 using dwb_core::CostmapROSPtr;
 
+#define NO_OP_DELETER [](auto){}
+
 namespace nav2_dwb_controller
 {
 
-// TODO(cdelsey): provide the correct clock to tfBuffer_
-DwbController::DwbController()
-: nav2_tasks::FollowPathTaskServer("FollowPathNode"), tfBuffer_(std::make_shared<rclcpp::Clock>()), tfListener_(tfBuffer_)
+DwbController::DwbController(rclcpp::executor::Executor & executor)
+: nav2_tasks::FollowPathTaskServer("FollowPathNode"), tfBuffer_(get_clock()), tfListener_(tfBuffer_)
 {
+  cm_ = std::make_shared<nav2_costmap_2d::Costmap2DROS>("local_costmap", tfBuffer_);
+  executor.add_node(cm_);
+  odom_sub_ = std::make_shared<nav_2d_utils::OdomSubscriber>(*this);
+  vel_pub_ = 
+    this->create_publisher<geometry_msgs::msg::Twist>("/mobile_base/commands/velocity", 1);
```

Comment:
Let's change this topic to "cmd_vel" to match the ROS topic

Reply:
Done